tests/test_serializers.py

```diff
@@ -0,0 +1,39 @@
+import pickle
+import sys
+
+import pytest
+from django.core.exceptions import ImproperlyConfigured
+
+from django_redis.serializers.pickle import PickleSerializer
+
+
+class TestPickleSerializer:
+    @pytest.mark.skipif(
+        sys.version_info < (3, 8),
+        reason="Protocol 5 was added as highest from python version 3.8",
+    )
+    def test_using_negative_protocol_value(self):
+        serializer_p5 = PickleSerializer({"PICKLE_VERSION": pickle.HIGHEST_PROTOCOL})
```

Comment:
better to specify version 5 so that it will keep passing even when there will an eventual version 6 in the future
```suggestion
        serializer_p5 = PickleSerializer({"PICKLE_VERSION": 5})
```

Reply:
-1 is always the highest protocol, but I'm a little confused why there's a skip for < Python 3.8. This should be valid for all Python versions and it's fine to just run the test everywhere instead of special casing this.

Reply:
I missed it, you are right, but then the assertion should be on the pickle_version used or the test does not make sense

Reply:
Yeah, I agree. I'm fine removing the test since it's actually just tesitng that -1 is interpreted as `HIGEST_VERSION` by `pickle` and that's outside the domain of this package.